test/integration/use_int_test.go

```diff
@@ -42,8 +42,12 @@ func (suite *UseIntegrationTestSuite) TestUse() {
 	cp.ExpectExitCode(0)
 
 	// Verify runtime works.
-	pythonExe := filepath.Join(ts.Dirs.DefaultBin, "python3"+osutils.ExeExt)
-	cp = ts.SpawnCmd(pythonExe, "--version")
+	pythonExe := filepath.Join(ts.Dirs.DefaultBin, "python3")
```

Comment:
If the intent is to return to the .bat/.sh executors, I believe that lines like this need code that looks more like: https://github.com/ActiveState/cli/blob/508375609b0c87794c7885a71f71c26fad5a482d/test/integration/use_int_test.go#L29-L32